From 361e659e70ffa4ac2662695aa46a01bc98913dea Mon Sep 17 00:00:00 2001
From: Richard Su <rwsu@redhat.com>
Date: Thu, 24 Apr 2014 15:28:04 -0700
Subject: [PATCH] Fix /var/lib/mysql selinux labeling

/var/lib/mysql is currently set to var_lib_t and should be
mysqld_db_t. restorecon will fix the file context.

Change-Id: Idf613fc0b9e1d218061057c0dad29929aded38bd
---
 .../mysql-common/os-refresh-config/pre-configure.d/97-mysql-selinux    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/elements/mysql-common/os-refresh-config/pre-configure.d/97-mysql-selinux b/elements/mysql-common/os-refresh-config/pre-configure.d/97-mysql-selinux
index 26d1170..9031636 100755
--- a/elements/mysql-common/os-refresh-config/pre-configure.d/97-mysql-selinux
+++ b/elements/mysql-common/os-refresh-config/pre-configure.d/97-mysql-selinux
@@ -14,3 +14,6 @@ restorecon -Rv /mnt/state/var/log/mysql
 [ -d /mnt/state/etc/mysql ] || install -m 0755 -o mysql -g mysql -d /mnt/state/etc/mysql
 semanage fcontext -a -t mysqld_etc_t "/mnt/state/etc/mysql(/.*)?"
 restorecon -Rv /mnt/state/etc/mysql
+
+# /var/lib/mysql needs to be set to mysqld_db_t
+restorecon -R /var/lib/mysql
-- 
1.9.3

